charts/posit-chronicle/README.md.gotmpl

```diff
@@ -0,0 +1,120 @@
+{{ template "rstudio.header" . }}
+
+{{ template "chart.versionBadge" . }}{{ template "chart.typeBadge" . }}{{ template "chart.appVersionBadge" . }}
+
+{{ template "rstudio.description" . }}
+
+{{ template "rstudio.disclaimer" . }}
+
+{{ template "rstudio.install" . }}
+
+## Usage
+
+This chart deploys only the Chronicle server and is meant to be used in tandem
+with the Workbench and Connect charts. To actually send data to the server, you
+will need to run the Chronicle agent as a sidecar container on all of your
+Workbench or Connect server pods by setting `pod.sidecar` in their respective
+`values.yaml` files.
+
+Here are some example Helm values to run the agent sidecar in Workbench:
+
+```yaml
+pod:
+  sidecar:
+    - name: chronicle-agent
+      image: posit-chronicle:latest
```

Comment:
It feels like putting the latest tag in the example might not be a great idea, could we template it to use the `appVersion` instead like our other helm charts?

Reply:
We can totally pin this to a non-lates image 👍 . Weversion the agent and server on a separate cadence, but probably can enforce this 1:1 versioning as a policy.
We've been thinking of migrating them both into 1 codebase, that would probably help too. This is one more reason to do that